src/ethereum/fees/ethMiningFees.js

```diff
@@ -34,10 +34,14 @@ export function calcMiningFee(
     const { customNetworkFee } = spendInfo || {}
     if (spendInfo.networkFeeOption === ES_FEE_CUSTOM && customNetworkFee) {
       const { gasLimit, gasPrice } = customNetworkFee
-      const {
-        gasLimit: { minGasLimit },
-        gasPrice: { minGasPrice }
-      } = currencyInfo.defaultSettings.otherSettings.defaultNetworkFees.default
+      const minGasLimit =
+        networkFees.default.gasLimit.minGasLimit ??
```

Comment:
Shouldn't you use more question marks incase something is undefined like this:
`networkFees?.default?.gasLimit?.minGasLimit` ?